@kiva/kv-components/src/vue/KvCarousel.vue

```diff
@@ -48,7 +48,24 @@
 				:disabled="embla && !embla.canScrollPrev()"
 				@click="handleUserInteraction(previousIndex, 'click-left-arrow')"
 			>
+				<span
+					v-if="roundedChevron"
+					class="tw-inline-flex tw-w-4"
```

Comment:
Something about these styles is not properly centering the new svg. It's visually off center within the circle.